### DISCUSSION TITLE
Handlebars not interpreting iframe content

### DISCUSSION THREAD

**Discussion by sdonnet** [handlebars-lang/handlebars.js#2036] [Q&A]
Hi,

I have an HTML document, with handlebars, partials, ... working well. "Handlebars" do the job.

But, I need to include into my HTML document several <iframe>, to separate different CSS stylesheets, which can interfere among them. So, I write something like :
`<iframe width="100%" src="data:text/plain;base64,{{> (lookup character 'chr_partial64') }}"></iframe>`

"character" being a JSON object, chr_partial64 a partial with base64 encoded value.

When rendering, the <iframe> is containing the HTML decoded from base64 data, but it is not interpreted by handlebars (the iframe HTML still contains handlebars tags (each, if, ...) not interpreted.

Is there a way to tell handlebars to interpret ? (I tested the triple mustaches, but unsuccessfully..

**Comment by jaylinski** [comment]
I think this is not an issue of handlebars, but has something to do with your setup.

A minimal reproduction of the issue would be very helpful. You can start here:
https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%3Ciframe%20width%3D%5C%22100%25%5C%22%20src%3D%5C%22data%3Atext%2Fplain%3Bbase64%2C%7B%7B%3E%20(lookup%20foo%20'bar')%20%7D%7D%5C%22%3E%3C%2Fiframe%3E%5Cn%22%2C%22partials%22%3A%5B%7B%22name%22%3A%22partial%22%2C%22content%22%3A%22test%22%7D%5D%2C%22input%22%3A%22%7B%5Cn%20%20foo%3A%20%7B%5Cn%20%20%20%20bar%3A%20%5C%22partial%5C%22%5Cn%20%20%7D%5Cn%7D%5Cn%22%2C%22output%22%3A%22%3Ciframe%20width%3D%5C%22100%25%5C%22%20src%3D%5C%22data%3Atext%2Fplain%3Bbase64%2Ctest%5C%22%3E%3C%2Fiframe%3E%5Cn%22%2C%22preparationScript%22%3A%22%22%2C%22handlebarsVersion%22%3A%224.7.8%22%7D
